styles/main.css

```diff
@@ -483,11 +484,11 @@ header nav input {
     }
 
     label.sandwich-icon::before {
-        top: 10px;
+        top: 5px;
     }
 
     label.sandwich-icon::after {
-        top: 30px;
+        top: 15px;
```

Comment:
Changing these values to get your X working made your main sandwich icon incorrect